src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/MicrosoftNetCoreAnalyzersResources.resx

```diff
@@ -1509,6 +1509,15 @@
   <data name="DoNotGuardDictionaryRemoveByContainsKeyTitle" xml:space="preserve">
     <value>Unnecessary call to 'Dictionary.ContainsKey(key)'</value>
   </data>
+  <data name="DoNotGuardSetAddOrRemoveByContainsDescription" xml:space="preserve">
+    <value>Do not guard 'Set.Add(item)' or 'Set.Remove(item)' with 'Set.Contains(item)'. The former two already check whether the item exists and will return if it was added or removed.</value>
```

Comment:
Using `'Set.Add(item)', 'Set.Remove(item)', 'Set.Contains(item)'` in the messaging here and below doesn't look right. I understand the issue written like this to illustrate the Sets in general, but the Sets flagged here are not derived from `Set`, they derive from <ISet> which doesn't have `Remove/Contains` methods, so it might cause a confusion